e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -592,6 +578,41 @@ if ($groupEnrollmentExists)
 Write-Host "`nAdding group enrollment $groupEnrollmentId."
 az iot dps enrollment-group create -g $ResourceGroup --dps-name $dpsName --enrollment-id $groupEnrollmentId --ca-name $dpsUploadCertificateName --output none
 
+$groupEnrollmentId = "Group1"
+$groupEnrollmentExists = az iot dps enrollment-group list -g $ResourceGroup --dps-name $dpsName --query "[?enrollmentGroupId=='$groupEnrollmentId'].enrollmentGroupId" --output tsv
+if ($groupEnrollmentExists)
+{
+    Write-Host "`nDeleting existing group enrollment $groupEnrollmentId."
+    az iot dps enrollment-group delete -g $ResourceGroup --dps-name $dpsName --enrollment-id $groupEnrollmentId
+}
+Write-Host "`nAdding group enrollment $groupEnrollmentId."
+az iot dps enrollment-group create -g $ResourceGroup --dps-name $dpsName --enrollment-id $groupEnrollmentId --ca-name $dpsUploadCertificateName --output none
```

Comment:
It looks like this section was duplicated. This code is already implemented above in L571-579.